x-pack/plugins/session_view/public/components/detail_panel_list_item/styles.ts

```diff
@@ -20,11 +20,14 @@ export const useStyles = ({ display }: StylesDeps) => {
     const item: CSSObject = {
       display,
       alignItems: 'center',
-      padding: euiTheme.size.s,
+      padding: `0px ${euiTheme.size.s} `,
       width: '100%',
-      fontSize: 'inherit',
+      fontSize: '14px',
```

Comment:
we need to avoid hardcoding font sizes. If design is not using what is provided by EUI in terms of sizing, they should align their designs with it.

Reply:
cc @codearos

Reply:
though looking at this in on my local. if you just remove fontSize, fontWeight and minHeight, from the item: CSSObject it seems to default to 14px which seems to be the the actual base size? e.g 1rem.